#If updated, ensure simple_task_executor_node.py/validate_types() is updated accordingly
version: 1.2.2    # This is the latest version ID
use_sim: BOOLEAN  # is this a simulated environment

#TOPOLOGY (Node Descriptors)
special_nodes:
  - id: NODE_ID
    descriptors:
      - base_station
      - wait_node
      - charging_station

#ROUTING
planning_type: STR # method used for coordinating routes

#INITIALISATION
agent_setups:
  - setup: &setup_00
    tasks:
      - module: transportation                  #Task module
        role: courier                           #Agents role in task
      - module: uv
        role: phototherapist
    has_presence: true
    proerties:
        key: value

agent_list:
  - agent_id: thorvald_001
    setup: *setup_01
  - agent_id: thorvald_002
    setup: *setup_01
  - agent_id: picker01
    setup: *setup_02
  - agent_id: picker02
    setup: *setup_02
  - agent_id: storage01
    setup: *setup_03
    initial_location: WayPoint69     #OPTIONAL
    
active_tasks:
  - module: transportation
    properties:
      wait_loading: 20
      wait_unloading: 40
  - module: uv
#  - module: low_battery
#    properties:
#      low_battery_voltage: 0.45

    

